turbinia/evidence.py

```diff
@@ -214,6 +214,21 @@ def __str__(self):
   def __repr__(self):
     return self.__str__()
 
+  @property
+  def name(self):
+    if self._name:
+      return self._name
+    else:
+      return self.source_path if self.source_path else self.type
```

Comment:
I wonder if we should make this something like `'{0:s}:{1:s}'.format(self.type, self.source_path)` so that it has both when that's available?

Reply:
(or just a `join()` like you have below)

Reply:
As discussed, I think I'll leave the type out for now (unless there's no other name to use). If this causes confusion, we can always add it in later.